src/resource.js

```diff
@@ -43,6 +43,7 @@ var Resource = function() {
   this.$$links = {};
   this.$$embedded = {};
   this.$$forms = {};
+  this.$$formatSpecific = {};
```

Comment:
And since it will be a "published" property, please add the JSDocs above the declaration to document its usage.